gson/src/main/java/com/google/gson/GsonBuilder.java

```diff
@@ -488,6 +492,17 @@ public GsonBuilder setPrettyPrinting() {
     return this;
   }
 
+  /**
+   * Configures Gson to output JSON that uses a certain kind of newline.
+   * This option only affects JSON serialization.
```

Comment:
Would be good to clarify that this has no effect unless pretty printing (with link to `#setPrettyPrinting`) is used. Or should this implicitly activate pretty printing (if so that would have to be documented here)?

Reply:
Added a more generic description: _"Has no effect if the serialized format is a single line"_

So that it does not depend on the exact settings, which might change.
For example right now `Gson().tojson("foo")` is one line, pretty-print or not.
But if someone adds a `setTrailingNewline()` setting 2 years from now this comment would still be true.